litestar/_kwargs/extractors.py

```diff
@@ -145,11 +147,13 @@ def parse_connection_query_params(connection: ASGIConnection, kwargs_model: Kwar
     Returns:
         A dictionary of parsed values.
     """
-    parsed_query = connection.scope["_parsed_query"] = (  # type: ignore
+    parsed_query = cast(
```

Comment:
this is a bit of a hot path, should we prefer using `# type: ignore` over `cast()` in these cases?

Reply:
changed it to the comment